layouts/shortcodes/pages/jakartaeechina/index.zh.html

```diff
@@ -0,0 +1,34 @@
+<h3 id="contact-us">Jakarta EE 中国社区</h3>
+<p>
+	Jakarta EE中国社区，是经Eclipse基金会授权，由普元信息（Primeton）发起创立并推动发展。致力于在大中化地区推动Jakarta EE的认知与技术分享。
+	<br>
+	正如我们所知道的，Java EE改名为Jakarta EE已有很长一段时间了，但是在中国，很少有人意识到Jakarta EE项目以及它的规范性实现的重要性。因此，普元信息现正呼吁您与我们一起努力，扩大Jakarta EE的影响力，为中国的开发者和中国企业做更多有益的事情。
+</p>
+
+<h3 id="out">Jakarta EE 中国社区邮件列表</h3>
+<p>订阅和了解最新的Jakarta EE 中国社区资讯</p> 
+<p class="margin-top-20"><a class="btn btn-lg btn-secondary" href="http://jakarta.eaworld.io/belisted/mailinglist.php">Jakarta EE 中国社区邮件列表</a></p>
+
+<h3 id="out">想要把你的名字也加入列表么？</h3>
+<p>目前Jakarta EE中国社区已经正式成立，我们欢迎所有对Java EE或Jakarta EE感兴趣的同学加入社区。</p> 
+<p class="margin-top-20"><a class="btn btn-lg btn-secondary" href="http://jakarta.eaworld.io/belisted">加入 Jakarta EE 中国社区</a></p>
+
+<h3 id="out">成员列表</h3>
+<p class="memberlist" style="line-height:2;"></p>
+
+<script src="https://apps.bdimg.com/libs/jquery/1.6.4/jquery.min.js" type="text/javascript"></script>
+
+<script>
+$.ajax({ 
+		type:"get",
+        dataType:"json",
+		url: "http://jakarta.eaworld.io/belisted/getlist.php", 
```

Comment:
Do we need to pull-in this data from an external domain?

Reply:
Since we can not reach the google form in China from time to time(just as now), we will need an online form registration.
Once someone has registered the form, we need to update his info on the members list on jakarta.ee/about/jakartachina page.
And because it is not so convenient to update the list timely, I think it is better to pull it from somewhere where things can be automately updated. For the time being , I deploy them on an external domain,  if these can be done on jakarta.ee, an external domain will not be needed.
Here I draw a picture( sorry for that I am not so good at it) to describe how it works. and due to my awful English, I'm not sure if I express everything clear. -_-
![image](https://user-images.githubusercontent.com/1784830/93352920-f09a7700-f86d-11ea-9bd4-bd2b0fbbc993.png)